editor/import/dynamic_font_import_settings.cpp

```diff
@@ -1403,7 +1403,7 @@ DynamicFontImportSettings::DynamicFontImportSettings() {
 
 	label_glyphs = memnew(Label);
 	gl_hb->add_child(label_glyphs);
-	label_glyphs->set_text(TTR("Preloaded glyphs:") + " " + itos(0));
+	label_glyphs->set_text(vformat(TTR("Preloaded glyphs: %d"), 0));
```

Comment:
Are there languages which would realistically need to move `%d` around here?
The `"Some text:" + " " + var` pattern is quite common throughout the codebase (I know the space after the `:` might not be the one you'd use in e.g. Chinese, but that's the most common pattern we use anyway).

Reply:
Including `%d` provides additional context. e.g. You know it's describing the number of glpyhs, not used as the label of a glphy list.
Weblate also complains about the translated string not including a trailing space but the original string has one.

Reply:
Yeah we should never have trailing spaces in msgids.

Reply:
We probably should use `vformat` everywhere, in some (rare) cases it's necessary to add control chars into the translation to ensure correct BiDi processing. Usually when it's something like `"Something something %s, %s and %s."` and inserted words are type names or something else that is not translated, but there might be other cases as well.